tests/palindrome_number_test.py

```diff
@@ -1,2 +1,34 @@
 import pytest
 from functions.palindrome_number import palindrome_number
+
+def test_returns_true_for_99():
+    # Arrange
+    n = 99
+
+    # Act
+    answer = palindrome_number(99)
+
+    # Assert
+    assert answer == True
```

Comment:
This looks pretty good. Here's another try:
'''python
assert answer == False
pass'''